source/renderer/app/api/etc/sendEtcBugReport.js

```diff
@@ -41,9 +41,12 @@ export const sendEtcBugReport = (
     path: '/report',
     port: reportUrl.port,
   }, {
-    application,
-    version: environment.version,
-    build: environment.build,
+    product: 'Cardano Wallet',
```

Comment:
@daniloprates I belive we should use "Mantis Wallet" as a product name for ETC bug reports.